src/shogun/evaluation/MachineEvaluation.cpp

```diff
@@ -121,5 +165,19 @@ CMachine* CMachineEvaluation::get_machine() const
 
 EEvaluationDirection CMachineEvaluation::get_evaluation_direction()
 {
-	return m_evaluation_criterion->get_evaluation_direction();
+    REQUIRE(m_list_evaluation_criterion->get_array_size()==1,"Multiple Metrics provided. Please use get_evaluation_directions()");
```

Comment:
I think I shouldn't `cout` that the user is supposed to use the other method, since he has provided multiple `CEvaluation`. Overloading the return type is not possible. If not `REQUIRE`, how should I use to tell the user to use the other method.